### PR TITLE
Add akka metrics grafana dashboard

### PR DESCRIPTION
**Pull request by GoutamVerma**
In this PR I have added `akka-metrics.json` file to monitor akka metrics in grafana dashboard. 

- Channels (4 panels)
channel

- Register (4 panels)
register

- Peers (4 panels)
peers

### CODE REVIEW

**Comment by t-bast** [comment]
The akka metrics we're interested in are:

- `akka.group.time-in-mailbox` -> this is the time spent before handling a message, it should always stay small
- `akka.group.processing-time` -> this is the time it takes to handle a message, it should also always stay small

You'll see that these metrics are emitted for every actor in the system, by using the `group` tag. You should filter this metric for specific `group` values depending on what actor you want to monitor. The actors we want to monitor for this first version are:

- the `Peer` actor: `group = eclair-node/user/SimpleSupervisor/Switchboard/Peer`
- the `Channel` actor: `group = eclair-node/user/SimpleSupervisor/Switchboard/Peer/Channel`
- the `Register` actor: `group = eclair-node/user/SimpleSupervisor/Register`

Here is a screenshot of what we currently have with Kamon for the `Peer` actor:

akka-metrics

Notice that for each of the metrics, we create **two** graphs to display it in two different ways:

-  a heatmap summarizing the distribution of values
- a percentile view (you will need to learn how to query for percentiles in prometheus and grafana, which is an important exercise - just search online for documentation and tutorials on how to do that)

**Comment by GoutamVerma** [comment]
> * a heatmap summarizing the distribution of values

I have create a two heatmap for each actors(time in mailbox, processing-time) and also added it in PR. 

> * a percentile view (you will need to learn how to query for percentiles in prometheus and grafana, which is an important exercise - just search online for documentation and tutorials on how to do that)

To view percentile in grafana and prometheus, promql has query `quantile_over_time`. I used this query and generate below panels for histogram and timeseries.

Query: `quantile_over_time(0.95rate(akka_group_time_in_mailbox_seconds_count{group="eclairnode/user/SimpleSupervisor/Switchboard/Peer/Channel"}[5m])[$__range:5m])`

Histogram:
histogram

Timeseries:
timeseries

More info about [percentile in grafana and prometheus](https://community.grafana.com/t/percentile-95-with-grafana-and-prometheus/30825/2)

File: monitoring/grafana-dashboard/akka-metrics.json [hunk 1]
```diff
@@ -0,0 +1,960 @@
...
+        }
+      },
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${DS_PROMETHEUS}"
+          },
+          "editorMode": "code",
+          "expr": "quantile_over_time(0.99,\n  rate(akka_group_time_in_mailbox_seconds_count{group=\"eclair-node/user/SimpleSupervisor/Switchboard/Peer/Channel\"}\n[6m])[$__range:6m]\n)",
```
Comment (t-bast) [inline monitoring/grafana-dashboard/akka-metrics.json:209]: You use the 99th percentile in some places and the 95th percentile in others, you should make it consistent.
Just use the 95th percentile everywhere, this is more interesting if we display only one percentile, and update the graph titles to explicitly say that they're displaying the 95th percentile.

Comment (GoutamVerma) [reply monitoring/grafana-dashboard/akka-metrics.json]: I have added new commits to correct percentile from 99th to 95th

**Comment by t-bast** [comment]
Let's merge this as-is, we can always improve it later. I strongly encourage node operators who want to experiment with these new monitoring dashboards to challenge the values they see and contribute back by enriching the graphs or fixing what may be incorrect.

**Review by t-bast — APPROVED** [review]

**Pull request merged**